ps_checkout.php

```diff
@@ -360,9 +363,15 @@ public function hookDisplayFooterProduct($params)
 
     public function getContent()
     {
+        $webhookHelper = new PrestaShop\AccountsAuth\Wrapper\Webhook();
+        $webhookHelper->ensureWebhookRegistered($this);
```

Comment:
Sert à enregistrer le Hook meme après l'install, pour faire fonctionner PS Checkout legacy correctement.

Reply:
As we don't know what is the time needed to run it each time we load the page, it could be enough to move this code in a upgrade file.